debug_toolbar/decorators.py

```diff
@@ -5,9 +5,17 @@
 
 from debug_toolbar import settings as dt_settings
 
+try:
+    from django.contrib.auth.decorators import login_not_required
+except ImportError:
+
+    def login_not_required(func):
+        return func
+
 
 def require_show_toolbar(view):
     @functools.wraps(view)
+    @login_not_required
```

Comment:
I don't think `require_show_toolbar` necessarily implies `login_not_required`. In practice it does, but I'd prefer if the decorator was added to the view function themselves, even though you have to scatter the `@login_not_required` invocations over the code base. 

I have added `authlib/_compat.py` (with a leading underscore to signify that this is purely internal and shouldn't be relied on), the same thing may make sense here.

Reply:
changed the implementation as suggested